specification/datamigration/resource-manager/Microsoft.DataMigration/preview/2018-07-15-preview/definitions/Projects.json

```diff
@@ -0,0 +1,151 @@
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {},
+  "definitions": {
+    "Project": {
+      "type": "object",
+      "description": "A project resource",
+      "properties": {
+        "properties": {
+          "description": "Project properties",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ProjectProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./Common.json#/definitions/TrackedResource"
+        }
+      ]
+    },
+    "ProjectArtifactsResponse": {
+      "type": "object",
+      "description": "Project artifacts properties",
+      "properties": {
+        "artifactsLocation": {
+          "type": "string",
+          "description": "A URL that points to the drop location to access project artifacts. When accessing this URL, requestor should provide an Authorization header with the Bearer authorization scheme, followed by his ARM JWT signature. Typical JWT will look like three dot-separated base64 encoded strings. We use last string as a shared secret between client and our artifacts service.\n"
+        }
+      }
+    },
+    "ProjectList": {
+      "type": "object",
+      "description": "OData page of project resources",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "List of projects",
+          "items": {
+            "$ref": "#/definitions/Project"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "URL to load the next page of projects"
+        }
+      }
+    },
+    "ProjectProperties": {
+      "type": "object",
+      "description": "Project-specific properties",
+      "properties": {
+        "sourcePlatform": {
+          "description": "Source platform for the project",
+          "$ref": "#/definitions/ProjectSourcePlatform"
+        },
+        "targetPlatform": {
+          "description": "Target platform for the project",
+          "$ref": "#/definitions/ProjectTargetPlatform"
+        },
+        "creationTime": {
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true,
+          "description": "UTC Date and time when project was created"
+        },
+        "sourceConnectionInfo": {
+          "$ref": "./Common.json#/definitions/ConnectionInfo",
+          "description": "Information for connecting to source"
+        },
+        "targetConnectionInfo": {
+          "$ref": "./Common.json#/definitions/ConnectionInfo",
+          "description": "Information for connecting to target"
+        },
+        "databasesInfo": {
+          "type": "array",
+          "description": "List of DatabaseInfo",
+          "items": {
+            "$ref": "#/definitions/DatabaseInfo"
+          }
+        },
+        "provisioningState": {
```

Comment:
Shouldn't there be something about online/offline migration in project properties?

Reply:
Not anymore.  Online/Offline is not associated with project but rather activity.